test/e2e/flowvisibility_test.go

```diff
@@ -462,6 +464,90 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 			checkRecordsForFlows(t, data, podAIPs.ipv4.String(), svcC.Spec.ClusterIP, isServiceIPv6, false, true, false, false)
 		}
 	})
+
+	// ClickHouseMonitor tests ensure ClickHouse monitor inspects the ClickHouse
+	// Pod storage usage and deletes the data when the stoage usage grows above
+	// the threshold.
+	t.Run("ClickHouseMonitor", func(t *testing.T) {
+		var flow testFlow
+		if !isIPv6 {
+			flow = testFlow{
+				srcIP:      podAIPs.ipv4.String(),
+				dstIP:      podBIPs.ipv4.String(),
+				srcPodName: "perftest-a",
+				dstPodName: "perftest-b",
+			}
+		} else {
+			flow = testFlow{
+				srcIP:      podAIPs.ipv6.String(),
+				dstIP:      podBIPs.ipv6.String(),
+				srcPodName: "perftest-a",
+				dstPodName: "perftest-b",
+			}
+		}
+		checkClickHouseMonitor(t, data, isIPv6, flow)
+	})
+}
+
+func checkClickHouseMonitor(t *testing.T, data *TestData, isIPv6 bool, flow testFlow) {
+	checkClickHouseMonitorLogs(t, data, false, 0)
+	var cmdStr string
+	if !isIPv6 {
+		cmdStr = fmt.Sprintf("iperf3 -u -c %s -P 128 -n 1", flow.dstIP)
```

Comment:
Better to have a comment here describing the choice of `128` as the max value for parallel clients.

Reply:
Added, thanks!